src/users.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {
+  List,
+  Datagrid,
+  TextField,
+  EmailField,
+  EditButton,
+  Edit,
+  SimpleForm,
+  TextInput,
+  useMutation,
+  Button
+} from "react-admin";
+
+const ApproveButton = ({ record }) => {
+  console.log(record);
+  const [approve, { loading }] = useMutation({
+    type: "update",
+    resource: "comments",
+    payload: { id: record.id, data: { isApproved: true } }
+  });
+  return <Button label="Approve" onClick={approve} disabled={loading} />;
```

Comment:
in here we need to check if it's already approved then show a text saying Approved

Reply:
i totally changed that in the new branch